probe-rs/src/coresight/memory/adi_v5_memory_interface.rs

```diff
@@ -166,19 +201,66 @@ impl ADIMemoryInterface {
         let tar = TAR { address };
         self.write_register_ap(debug_port, tar)?;
 
-        for data in data.iter_mut() {
-            *data = self.read_register_ap(debug_port, DRW::default())?.data;
+        // figure out how many words we can write before the
+        // data overflows
+
+        // maximum chunk size
+        let max_chunk_size_bytes = 0x400_usize;
```

Comment:
Is this the format of cargo fmt? Because I think `let max_chunk_size_bytes: usize = 0x400;` would be way better.